aten/src/ATen/native/native_functions.yaml

```diff
@@ -5836,6 +5836,10 @@
   dispatch:
     DefaultBackend: addcdiv
 
+- func: cross_entropy_loss(Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean, int ignore_index=-100) -> Tensor
+  use_c10_dispatcher: hacky_wrapper_for_legacy_signatures
```

Comment:
hacky_wrapper shouldn't be used for new ops, we are currently in the process of getting rid of it for legacy ops as well. In this specific case, you can remove hacky_wrapper if you make the implementation take c10::optional<Tensor> instead of Tensor for the optional weight argument.